components/clp-py-utils/clp_py_utils/initialize-spider-db.py

```
@@ -0,0 +1,302 @@
+                db_cursor.execute(
+                    f"""CREATE USER IF NOT EXISTS '{db_user}'@'%' IDENTIFIED BY '{db_password}'"""
+                )
+            else:
+                db_cursor.execute(f"""CREATE USER IF NOT EXISTS '{db_user}'@'%' IDENTIFIED BY ''""")
+            db_cursor.execute(f"""GRANT ALL PRIVILEGES ON `{db_name}`.* TO '{db_user}'@'%'""")
+            db_cursor.execute(f"""GRANT ALL PRIVILEGES ON `{db_name}`.* TO '{clp_user}'@'%'""")
+
+            db_cursor.execute(f"""USE `{db_name}`""")
+            for table_creator in table_creators:
+                db_cursor.execute(table_creator)
+    except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)